packages/terra-switch/src/terra-dev-site/doc/example/SwitchWithOnChange.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+import Switch from 'terra-switch';
+
+export default () => {
+  const [value, setValue] = useState(true); // makes switch on by default
+  return (
+    <div>
+      <Switch
+        id="defaultSwitchOn"
+        labelId="label3"
+        checked={value}
+        label="Label"
+        onChange={(event, checked) => { setValue(!checked); }}
```

Comment:
If the params are flipped and the onChange return value is updated the onChange can be simplified to:

```jsx
<Switch 
  id="defaultSwitchOn"
  labelId="label3"
  checked={value}
  label="Label"
  onChange={setValue}
/>
```

Reply:
[updated example](https://github.com/cerner/terra-core/commit/fcbd00f1ab0c8f611b576559dc12aaa9e5bd6173#diff-87a36fdb3c8438815231148acb245116)